packages/blockchain-website/src/components/IssuesList/IssuesList.tsx

```diff
@@ -0,0 +1,159 @@
+import React, { Component } from 'react';
+import './IssuesList.scss';
+
+const githubProject: string = 'IBM-Blockchain/blockchain-vscode-extension';
+
+// Github URLs
+const issuesHtmlUrl: string = 'https://github.com/' + githubProject + '/issues?q=';
+const newIssueUrl: string = 'https://github.com/' + githubProject + '/issues/new';
+
+interface IssueProps {
+    issues?: any;
+    error?: any;
+    title?: any;
```

Comment:
these still need some types `milestoneNumber` is surely a number or maybe a string